docs/release/cut-sw-release.md

```diff
@@ -352,7 +352,7 @@ The following instructions are meant for the release manager (or interim release
          
          We are pleased to announce OSG Software version <VERSION(S)>!
          
-         This is the new OSG Software distributed via RPMs for:
+         This is the OSG Software distributed via RPMs for:
```

Comment:
I don't think "new" here is referencing the fact that it's being distributed via RPM but rather that there's "new stuff" in the "new" version. I can go either way on removing it.

Reply:
I'd prefer to remove it. A release is always new.